onnxruntime/core/providers/cpu/math/gemm.cc

```diff
@@ -138,8 +166,70 @@
                 thread_pool);
 }
 
+template <>
+void Gemm<MLFloat16>::ComputeGemm(CBLAS_TRANSPOSE trans_a, CBLAS_TRANSPOSE trans_b,
```

Comment:
Consider specializing the compute function instead? It's kinda hard to fuse the activation function into the post processor this way.

Reply:
Search for MLAS_HALF_GEMM_ACTIVATION_PROCESSOR in fp16_conv.cc, for examples on how activation is computed together with MlasHalfGemmBatch

Reply:
Would prefer to do the activation fusion in the next PR.